inc/clean-emails.php

```diff
@@ -108,24 +111,39 @@ public function comment_notification_text( $message, $comment_id ): string {
 	public function comment_moderation_text( $message, $comment_id ): string {
 		$this->setup_data( $comment_id );
 
+		$comment_link = '<a href="' . \get_permalink( $this->comment->comment_post_ID ) . '">' . \esc_html( $this->post->post_title ) . '</a>';
```

Comment:
Should we `esc_url` the `get_permalink` here?

Reply:
Done